employee-onboarding-A2A-reference-implementation/agents/booking_agent_adk/__main__.py

```
@@ -0,0 +1,226 @@
+        token = self._current_request_token
+        if token and self._agent_id:
+            try:
+                from src.auth.asgardeo import get_asgardeo_client
+                from src.log_broadcaster import log_and_broadcast
+                asgardeo = get_asgardeo_client()
+                actor = await asgardeo._fetch_agent_actor_token(
+                    client_id=self._token_exchanger_client_id,
+                    client_secret=self._token_exchanger_client_secret,
+                    agent_id=self._agent_id,
+                )
+                log_and_broadcast(f"\n[BOOKING_AGENT_ACTOR_TOKEN]:")
```

Ruff (0.15.10)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)